generic/src/main/java/net/swofty/types/generic/event/actions/player/ActionPlayerBankAddInterest.java

```diff
@@ -38,16 +38,9 @@ public void run(Event tempEvent) {
             CoopDatabase.Coop coop = player.getCoop();
             coop.getMembersAsProxyPlayerSet(player.getUuid()).asProxyPlayers().forEach(ProxyPlayer::getBankHash);
         }
-
-        double totalEarnt = 0;
-        // Split up difference across INTEREST_INTERVAL, cast to int to get the amount of times interest should be added
-        int times = (int) (difference / SkyBlockCalendar.INTEREST_INTERVAL);
-        for (int i = 0; i < times; i++) {
-            totalEarnt += bankData.getAmount() * SkyBlockCalendar.INTEREST_RATE;
-        }
-
-        // Cap at bank limit
-        totalEarnt = Math.min(bankData.getAmount() + totalEarnt, bankData.getBalanceLimit() - bankData.getAmount());
```

Comment:
Doesn't this remove the part where it caps at the bank limit?

Reply:
Whoops yea I didn't mean to remove that and yes Hypixel caps it at 2